collator/src/lib.rs

```diff
@@ -208,8 +214,17 @@ where
 			})
 			.ok()?;
 
+		let validation_data = {
+			// TODO: Actual proof is to be created in the upcoming PRs.
```

Comment:
issue?

Reply:
Implementation for this literally sits in the next commit : )